packages/react/src/CTABanner/CTABanner.module.css

```
@@ -0,0 +1,76 @@
+.CTABanner {
+  position: relative;
+  max-width: 1280px;
+  margin: 0 auto;
+  display: flex;
+  gap: var(--base-size-16);
+  flex-direction: column;
+}
+
+.CTABanner--Shadow {
+  position: absolute;
+  transform: translate(-50%, 0%);
+  width: calc(60% + 60px);
+  height: calc(100% + 60px);
+  border-radius: 50% 22% 40% 80%;
+  filter: blur(50px);
+  top: 0%;
+  opacity: 0.7;
+}
+
+.CTABanner--RightShadow {
+  left: 30%;
+  background: radial-gradient(circle at 50% 50%, rgba(26, 127, 55, 1), rgba(26, 127, 55, 0));
+}
+
+.CTABanner--LeftShadow {
+  left: 70%;
+  background: radial-gradient(circle at 50% 50%, rgba(3, 102, 214, 1), rgba(3, 102, 214, 0));
+}
+
+.CTABanner--content {
+  border-radius: 6px;
+  padding: 40px 32px;
+  z-index: 1;
+  background: white;
+}
+
+.CTABanner--content--noshadow {
```

Is there an extra dash here?

```suggestion
.CTABanner-content--noshadow {
```